docs/Fabulous.XamarinForms/views-perf.md

```diff
@@ -47,6 +47,39 @@ You can also use
 * the `fix` function for portions of a view that have no dependencies at all (besides the "dispatch" function)
 * the `fixf` function for command callbacks that have no dependencies at all (besides the "dispatch" function)
 
+### View keys 101 
+  1. What is view key? 
+      View key is a little helper property like in this sample: 
+      
+     ```fsharp
+         // Previous View
+         View.Grid([
+                 View.Label(key = "header", text = "Old Header")
+                 View.Label(key = "body", text = "Old Body")
+             ])
+         
+         // New View
+         View.Grid([
+             View.Label(key = "header", text = "New Header") // Will reuse previous header
+             View.Button(key = "body", text = "New body") // Won't be able to reuse previous body since Label != Button
+         ])
+     ```
+         
+  As we can see here, we have added a 'key' parameter to each Label.
+  Key should be unique across view elements in same collection, i.e.
+  Children of StackLayout and so on.
+  
+  2.What is a purpose of key? 
+   Purpose of key is following: 
+  * make determination of proper view element to replace easier(just by key,yes)
+  * most important,to keep a reference to previous XF control(so we would not create it, but reuse existing).
+  
+  In many scenarios this is essential thing, for example, you draw a control
+  that streams video, and its position changed due some conditions.Without 'key' set,
+  we get continuous recreating of control with a bunch of issues like video interrupt.
+  With key, we continue reuse  same control on each view cycle.
```

Comment:
We should say the opposite.

In the majority of scenarios, Fabulous will be efficient without doing anything specific.
In some advanced scenarios, like the one above, `key` will let you help Fabulous be even more efficient.